client/src/containers/routing/index.tsx

```diff
@@ -44,13 +44,20 @@ const Routing: React.FC<IRoutingProps> = () => {
 			<PublicRoute exact restricted={true} path={ROUTES.Landing} component={LandingPageCointainer} />
 			<PrivateRoute path="/private" component={TestPrivate} needHeader={false} needSideBar={false} />
 			<PrivateRoute exact path={ROUTES.Home} component={HomePage} needHeader={true} needSideBar={true} />
+			<PrivateRoute
+				exact
+				path={ROUTES.createTask}
+				component={CreateTaskPage}
```

Comment:
here I think would be great to use same logic and use word with capital letter like ROUTES.CreatTask or ROUTES,NewTask